client/my-sites/checkout/cart/cart-item.jsx

```diff
@@ -63,6 +65,28 @@ export class CartItem extends React.Component {
 			return <span className="cart__free-text">{ translate( 'Free' ) }</span>;
 		}
 
+		if ( isGSuiteProductSlug( cartItem.product_slug ) ) {
+			const { cost_before_coupon: costBeforeCoupon } = cartItem;
+
+			if ( costBeforeCoupon ) {
```

Comment:
We should revisit that condition. I've indeed encountered the case where the API would return a value for this property even if no coupon was applied.

Reply:
This also doesn't handle very well the case where the user enters a coupon.

Reply:
I think I found a way to address my first comment. I'm still looking how to handle multiple coupons.

Reply:
This should be fixed with the `is_sale_coupon_applied` property in D36169-code. It's more reliable to depend on that.